client/errors.go

```diff
@@ -6,6 +6,9 @@ import (
 )
 
 var (
+	ErrMissingServer      = errors.New("server address is unset or empty")
+	ErrNonPositiveTimeout = errors.New("non-positive timeout")
+
```

Comment:
The best way to do these lists is:
 * alphabet sorted
 * without blank lines

Not really critical, but when you can (a new one is created), better do it this way.

Reply:
ok